src/Microsoft.Azure.WebJobs.Logging/Entities/InstanceTableEntity.cs

```diff
@@ -90,9 +89,14 @@ internal static string RowKeyFunctionInstanceId(Guid functionInstanceId)
 
         public string TriggerReason { get; set; }
 
-        public DateTimeOffset StartTime { get; set; }
+        public DateTime StartTime { get; set; }
```

Comment:
Also note this is only changing the type (DateTimeOffset --> DateTime) , which have the same server representation.  It doesn't change the field name. 

---
In reply to: [90368085](https://github.com/Azure/azure-webjobs-sdk/pull/928#discussion_r90368085) [](ancestors = 90368085)